lib/diagnostic.rb

```diff
@@ -13,28 +13,29 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce do |sum, n|
+      sum + n
+    end
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    numbers.even?
```

Comment:
What you want to be doing here is checking each number and running `.even?` on it. 
In particular, since you want to check if every number is even, you could write:
`numbers.all? { |num| num.even? }` (could also be written as `numbers.all?(&:even?)`)